pkg/wasm/runtime/wazero/wazero.go

```diff
@@ -1,6 +1,3 @@
-//go:build wasmer
-// +build wasmer
-
 /*
```

Comment:
by default, we should let both wazero and wasmer to be built in

Reply:
wazero doesn't need a build tag because has no deps. but wasmer used to have a build tag because it has system library and portability issues. so you want to have a build tag for wazero knowing this?

Reply:
having both in main is here now. what you are asking here is to make sure there's no engine available unless a user adds a build tag. I think this is a mistake as it will ensure tests don't run as the fake isn't good enough to test anything. This leaves one question and if you insist I can disable even wazero by default, just be prepared for things to be broke and no one notice.

Reply:
i think there is somewhat misunderstanding, my first thought was to keep both vms built in, not `there's no engine available unless a user adds a build tag`. And we allow to just build in wazero in the default make target `build-local`, but at the same time we should allow user to turn on wasmer by simple way just like adding a new make target `build-local-wasmer`, in other words, we do not wishes that our users are required to do any code modification (include `import`) to turn on wasmer

Reply:
ok so what's left is user and build instructions.

With current commit the docker build adds by default wasmer, because before it also had the build tag set for it.

for build-local, there's the choice to reduce that to the platform choices of wasmer 1.04, by adding the tag there also.

There's also the choice to show users how to use the existing TAGS variable, and finally to do that for them with a new target which adds that. Which do you prefer?

Reply:
I guess what I'm saying is.. before build-local didn't build in wasmer as far as I can tell, even if docker did. If you want the change to add wazero to also include build local for wasmer, we can do that. However, before it didn't except when someone set the build tag wasmer via TAGS.

Reply:
ex in 1.2.0 you can see this, which means wasmer doesn't build unless you added it via TAGS, which docker does and build-local doesn't.

```
pkg/wasm/runtime/wasmer/vm.go
1://go:build wasmer
2:// +build wasmer
```

Reply:
added it anyway :D